src/sparseml/transformers/token_classification.py

```diff
@@ -658,7 +658,7 @@ def compute_metrics(p):
         elif last_checkpoint is not None:
             checkpoint = last_checkpoint
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
-        if trainer.one_shot:
+        if not trainer.one_shot:
```

Comment:
can you do another comb for issues like this? LGTM after

Reply:
I checked all other scripts for such typos, it looks okay and the tests ran successfully